src/bluehawk/actions/ConsoleActionReporter.ts

```diff
@@ -85,7 +85,7 @@ export class ConsoleActionReporter implements ActionReporter {
     ++this._count.errors;
     if (this.logLevel >= LogLevel.Error) {
       console.error(
-        `failed to write file ${event.sourcePath} -> ${event.destinationPath}: ${event.error.message}`
+        `failed to write file ${event.sourcePath} -> ${event.outputPath}: ${event.error.message}`
```

Comment:
"source" -> "destination"
"input" -> "output"

So it might be nice to change `sourcePath` to `inputPath` everywhere as well...